src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace AsyncAws\Illuminate\Mail\Transport;
+
+use AsyncAws\Ses\SesClient;
+use AsyncAws\Ses\ValueObject\EmailContent;
+use Illuminate\Mail\Transport\Transport;
+use Swift_Mime_SimpleMessage;
+
+/**
+ * This class is a port from Illuminate\Mail\Transport\SesTransport.
+ */
+class AsyncAwsSesTransport extends Transport
+{
+    /**
+     * The Amazon SES instance.
+     *
+     * @var SesClient
+     */
+    protected $ses;
+
+    /**
+     * The Amazon SES transmission options.
+     *
+     * @var array
+     */
+    protected $options = [];
+
+    /**
+     * Create a new SES transport instance.
+     *
+     * @param array $options
+     *
+     * @return void
+     */
+    public function __construct(SesClient $ses, $options = [])
+    {
+        $this->ses = $ses;
+        $this->options = $options;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = null)
+    {
+        $this->beforeSendPerformed($message);
+
+        $input = [
+            'Content' => new EmailContent([
+                'Raw' => ['Data' => $message->toString()],
+            ]),
+        ];
+
+        $from = key($message->getSender() ?: $message->getFrom());
```

Comment:
getFrom returns mixed, `key()` expects an array
https://github.com/swiftmailer/swiftmailer/blob/4633950338d22b4406e5c77f6e25c658435d5db4/lib/classes/Swift/Mime/SimpleMessage.php#L223-L225

Reply:
Oh. That wasn't the psalm error I was fightning.. Hm.. Or maybe it was but the message was different.

Reply:
Could you help with this?
https://github.com/async-aws/aws/pull/550/checks?check_run_id=652813243#step:5:10

Reply:
.. I don't see the issue

Reply:
```
Note: Using configuration file /github/workspace/phpstan.neon.dist.
 ------ -------------------------------------------------------------------- 
  Line   Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php     
 ------ -------------------------------------------------------------------- 
  51     Parameter #1 $input of class AsyncAws\Ses\ValueObject\EmailContent  
         constructor expects array()|array('Simple' =>                       
         array|AsyncAws\Ses\ValueObject\Message|null, 'Raw' =>               
         array|AsyncAws\Ses\ValueObject\RawMessage|null, 'Template' =>       
         array|AsyncAws\Ses\ValueObject\Template|null), array('Raw' =>       
         AsyncAws\Ses\ValueObject\RawMessage) given.                         
 ------ -------------------------------------------------------------------- 
```